tensorflow/python/kernel_tests/init_ops_test.py

```diff
@@ -362,6 +362,70 @@ def testInvalidDataType(self):
         dtype=dtypes.string)
 
 
+class VarianceScalingInitializationTest(test.TestCase):
+
+  def testInitializerIdentical(self):
+    for dtype in [dtypes.float32, dtypes.float64]:
+      init1 = init_ops.variance_scaling_initializer(seed=1, dtype=dtype)
+      init2 = init_ops.variance_scaling_initializer(seed=1, dtype=dtype)
+      self.assertTrue(identicaltest(self, init1, init2))
+      init3 = init_ops.variance_scaling_initializer(
+          1.5, seed=1, dtype=dtype)
+      init4 = init_ops.variance_scaling_initializer(
+          1.5, seed=1, dtype=dtype)
+      self.assertTrue(identicaltest(self, init3, init4))
+
+  def testInitializerDifferent(self):
+    for dtype in [dtypes.float32, dtypes.float64]:
+      init1 = init_ops.variance_scaling_initializer(seed=1, dtype=dtype)
+      init2 = init_ops.variance_scaling_initializer(seed=2, dtype=dtype)
+      init3 = init_ops.variance_scaling_initializer(
+          1.5, seed=1, dtype=dtype)
+      self.assertFalse(identicaltest(self, init1, init2))
+      self.assertFalse(identicaltest(self, init1, init3))
+      self.assertFalse(identicaltest(self, init2, init3))
+
+  def testZeroSize(self):
+    shape = [0, 2]
+    with self.test_session():
+      x = variable_scope.get_variable(
```

Comment:
Please remove use of `get_variable` here too

Reply:
Should I change it everywhere while I'm at it? After all, I just copied the code for these tests...

Reply:
No need. And don't feel obligated to include tests like `testZeroSize` since it has effectively no connection to your fix.

Reply:
I am aware of that. On the other, hand I do think it is a good idea to have complete tests to avoid introducing new bugs...

PS: the commit will probably be for tomorrow